packages/system-upgrade-controller/generated-changes/patch/templates/configmap.yaml.patch

```diff
@@ -0,0 +1,17 @@
+--- charts-original/templates/configmap.yaml
++++ charts/templates/configmap.yaml
+@@ -2,13 +2,12 @@
+ kind: ConfigMap
+ metadata:
+   name: system-upgrade-controller-config
+-  namespace: cattle-system
```

Comment:
If we drop the namespace, won't this be created in the default namespace? If you want this in the namespace specified for the helm chart, this should be `{{ .Release.Namespace }}`